A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml

```diff
@@ -1232,36 +1245,6 @@
 
         <!-- Miscellaneous ==========================================================================================  -->
 
-        <Sound WwiseData="true" WwiseEvent="ptutest" NodeName="SOUND_ENGINE_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
-            <Requires SimVar="TURB ENG N1" Units="PERCENT" Index="1">
-                <Range UpperBound="1" />
-            </Requires>
-        </Sound>
-
-        <Sound WwiseData="true" WwiseEvent="ptutest" NodeName="SOUND_ENGINE_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
-            <Requires SimVar="TURB ENG N1" Units="PERCENT" Index="2">
-                <Range UpperBound="1" />
-            </Requires>
-        </Sound>
-        
-         <Sound WwiseData="true" WwiseEvent="ptucockpit" NodeName="PEDALS_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
-            <Requires SimVar="TURB ENG N1" Units="PERCENT" Index="1">
-                <Range UpperBound="1" />
-            </Requires>
-            <Requires LocalVar="A32NX_SOUND_PTU_AUDIBLE_COCKPIT" Units="BOOL">
-                <Range LowerBound="1" />
-            </Requires>
-        </Sound>
-
-        <Sound WwiseData="true" WwiseEvent="ptucockpit" NodeName="PEDALS_LEFT" LocalVar="XMLVAR_PTU_ON" Continuous="false">
-            <Requires SimVar="TURB ENG N1" Units="PERCENT" Index="2">
-                <Range UpperBound="1" />
-            </Requires>
-            <Requires LocalVar="A32NX_SOUND_PTU_AUDIBLE_COCKPIT" Units="BOOL">
-                <Range LowerBound="1" />
-            </Requires>
-        </Sound>
-
```

Comment:
Why did you remove these? Weren't these just added?

Reply:
they caused problems with the current ptu sound for some odd reason, so I removed and remade wwise events with a different name